code/game/gamemodes/changeling/powers/biodegrade.dm

```diff
@@ -60,6 +69,13 @@
 		O.forceMove(get_turf(user))
 		qdel(O)
 
+/datum/action/changeling/biodegrade/proc/dissolve_legcuffs(mob/living/carbon/human/user, obj/O)
+	if(O && user.legcuffed == O)
+		user.unEquip(O)
+		O.visible_message("<span class='warning'>[O] dissolves into a puddle of sizzling goop.</span>")
+		O.forceMove(get_turf(user))
+		qdel(O)
```

Comment:
`user.uncuff()` removes legcuffs as well as handcuffs, so you can probably add all of this to the `dissolve_handcuffs()` proc if you wanted. So long as you remember to delete them after being dropped.

Reply:
Instead of using `.uncuff()` I have made the dissolve procs for handcuffs/legcuffs/straightjacket into one more generic proc.